src/components/CircularOutline/CircularOutline.js

```diff
@@ -103,15 +103,15 @@ class RoundedOutline extends Component<Props, State> {
 
     const svgId = `${color1.replace('#', '')}-${color2.replace('#', '')}`;
 
-    const dashOffset = isShown ? 0 : pathLength;
+    const finalDashOffset = isShown ? 0 : pathLength;
 
     return (
       <Motion
         style={{
           dashOffset:
             animateChanges && finishedAllMountingSteps
-              ? spring(dashOffset, springSettings)
-              : dashOffset,
+              ? spring(finalDashOffset, springSettings)
+              : finalDashOffset,
```

Comment:
Yeah, I've been resistant to this as well, since in my mind it _is_ the same value, semantically; it's just that the shadowed one is interpolated.

But yeah, I can also see how that's ambiguous and not as beginner-friendly.

For the convention, what do you think about keeping the original variable (in this case, `dashOffset`), but making the sprung version `interpolatedDashOffset` or `sprungDashOffset`? Often the spring values come from props, so I think we should preserve the original variable's name, and rename the shadowed one instead.